Misc/NEWS.d/next/Library/2024-06-03-13-48-44.gh-issue-119933.Kc0HG5.rst

```diff
@@ -0,0 +1 @@
+Improve :exc:`SyntaxError` messages for :pep:`695` and :pep:`696`.
```

Comment:
Should also mention the changes to the symtable module, and reference the features by name instead of the PEPs.